### PR TITLE
SE-1392: add fx composite notebook

### PR DESCRIPTION
**Pull request by slemasne**

### CODE REVIEW

File: examples/use-cases/risk-and-performance/Composite returns adjusted for FX rates.ipynb [hunk 1]
```diff
@@ -0,0 +1,895 @@
+{
```
Comment (rizwansaeed) [inline examples/use-cases/risk-and-performance/Composite returns adjusted for FX rates.ipynb:1]: <p>is <code>default</code>  a special value or one that you've chosen? if it is a specific one can we add a note explaining</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (rizwansaeed) [inline examples/use-cases/risk-and-performance/Composite returns adjusted for FX rates.ipynb:1]: <p>what is this format, is there a KB article we can link to?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (rizwansaeed) [inline examples/use-cases/risk-and-performance/Composite returns adjusted for FX rates.ipynb:1]: <p>is <code>inc</code>  meant to be <code>inception</code>?  (characters are free...)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (rizwansaeed) [inline examples/use-cases/risk-and-performance/Composite returns adjusted for FX rates.ipynb:1]: <p>typo in <code>list_of_porfolios</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

**Comment by jyaffe** [comment]
It would be beneficial to add in commentary explaining why the relationship creation is necessary, as well as a pointer to where it comes into play when getting the returns back for the portfolios.

File: examples/use-cases/risk-and-performance/Composite returns adjusted for FX rates.ipynb (hunk 1 above)
Comment (slemasne) [reply examples/use-cases/risk-and-performance/Composite returns adjusted for FX rates.ipynb:1]: It is a specific one. I have added comment

Comment (slemasne) [reply examples/use-cases/risk-and-performance/Composite returns adjusted for FX rates.ipynb:1]: <p>Yes, we have a KB, I have linked it. </p>

Comment (slemasne) [reply examples/use-cases/risk-and-performance/Composite returns adjusted for FX rates.ipynb:1]: <p>Yes, inception. Updated. </p>

Comment (slemasne) [reply examples/use-cases/risk-and-performance/Composite returns adjusted for FX rates.ipynb:1]: <p>Thanks. Updated. </p>

**Comment by slemasne** [comment]
<p>Thanks Jack. Added comments to section 3.2</p>

**Review by rizwansaeed — APPROVED** [review]

**Pull request merged**